modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/network.scala

```diff
@@ -0,0 +1,52 @@
+package org.tessellation.rosetta.server.model
+
+import cats.Show
+import cats.implicits.toBifunctorOps
+
+import derevo.cats.show
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.predicates.all.NonEmpty
+import eu.timepit.refined.refineV
+import eu.timepit.refined.types.string.NonEmptyString
+import io.estatico.newtype.macros.newtype
+import io.getquill.MappedEncoding
+
+object network {
+  implicit def showNonEmptyString: Show[NonEmptyString] = s => s.value
+
+  @derive(show)
+  @newtype
+  case class NetworkStatus(value: NonEmptyString)
+
+  object NetworkStatus {
+    val accepted: NetworkStatus = NetworkStatus("Accepted")
+    val reverted: NetworkStatus = NetworkStatus("Reverted")
+    val canceled: NetworkStatus = NetworkStatus("Canceled")
```

Comment:
how network can be accepted, reverted or canceled?

Reply:
This denotes the status of a transaction on a network. So a transaction can be `Accepted`, `Reverted`, or `Canceled`.

Reply:
the name of this doesn't say that it's a status of a transaction on a network, but a status of the network itself.
also, is it 1:1 correspondence from rosetta? because transaction cannot be canceled or reverted in our network. eventually it can be not accepted

Reply:
Examples of usage are at [here](https://www.rosetta-api.org/docs/1.3.1/BlockApi.html#200---ok-1) and [here](https://www.rosetta-api.org/docs/1.3.1/MempoolApi.html#200---ok-1) where it is part of the transaction information returned. I will confer with @ryle-goehausen on the history of this class.

Reply:
Yes, it's used as a status of a transaction, not the network.

Reply:
So `NetworkStatus` is the `status` property [here](https://www.rosetta-api.org/docs/models/Operation.html). I will change the name to `TransactionOperationStatus`.

Reply:
:+1: